main.go

```diff
@@ -275,6 +287,124 @@ func cmdPrintConfig(ctx *cli.Context) error {
 	return nil
 }
 
+func cmdRegisterEstuary(ctx *cli.Context) error {
+	endpointURL := ctx.Args().Get(0)
+	token := ctx.Args().Get(1)
+
+	if endpointURL == "" {
+		return fmt.Errorf("an Estuary endpoint URL is required (first argument)")
+	}
+
+	if token == "" {
+		return fmt.Errorf("an Estuary admin token is required (second argument)")
+	}
+
+	// Get public IP address
+
+	publicIPRes, err := http.Get("http://ip-api.com/json")
```

Comment:
do we trust this response? Should the public IP be actually configured manually in the config file?

Reply:
we can modify this as necessary in the future. this is just a convenience tool and the worst thing that could happen is for an incorrect ip address to be advertised.